docs/source/en/api/pipelines/unidiffuser.md

```diff
@@ -20,6 +20,10 @@ The abstract from the [paper](https://arxiv.org/abs/2303.06555) is:
 
 You can find the original codebase at [thu-ml/unidiffuser](https://github.com/thu-ml/unidiffuser) and additional checkpoints at [thu-ml](https://huggingface.co/thu-ml).
 
+Tips:
+
+- There is currently an issue on PyTorch 1.X where the output images are all black or the pixel values become `NaNs`. This issue can be mitigated by switching to PyTorch 2.X.
+
```

Comment:
The doc-builder has a warning callout that is suited for a situation like this :)

```suggestion
<Tip warning={true}>

There is currently an issue on PyTorch 1.X where the output images are all black, or the pixel values become `NaNs`. This issue can be temporarily resolved by switching to PyTorch 2.X until a more permanent fix is implemented.

</Tip>
```

Reply:
Updated the docs to use the warning callout.